base/context.go

```diff
@@ -166,7 +167,10 @@ func (ctx *Context) GetHTTPClient() (*http.Client, error) {
 	if err != nil {
 		return nil, err
 	}
-	ctx.httpClient = &http.Client{Transport: &http.Transport{TLSClientConfig: tlsConfig}}
+	ctx.httpClient = &http.Client{
+		Transport: &http.Transport{TLSClientConfig: tlsConfig},
+		Timeout:   200 * time.Millisecond,
```

Comment:
this is too low. Talking to prod stuff on the other side of the country will get uncomfortably close to it.
It should also be a constant somewhere.